cmd/proxy/main.go

```diff
@@ -160,6 +163,11 @@ VERSION:
 		Name:  "start-swagger-ui",
 		Usage: "If set to true, will start a Swagger UI on the root",
 	}
+	// sovereignConfig defines a flag that specifies if what run type components should use
+	sovereignConfig = cli.BoolFlag{
```

Comment:
Let's rename it, please. We do not use/change any config.
Should be simpler to just use `sovereign`